components/artwork_item/templates/artwork.jade

```diff
@@ -19,6 +19,8 @@ figure.artwork-item( data-artwork= artwork.get('id') data-id= artwork.get('_id')
       include ./partials/save
 
   figcaption.artwork-item-caption
+    if hasAuctionPartner && !isAuction
```

Comment:
I wonder if we should update the logic we use to determine whether an artwork is part of an auction... now that artworks have `sale_ids` denormalized, might be better to check for that. I'm actually not sure what the `isAuction` check is looking for. Do we still use `sets` for these things?